src/components/BulkEnrollmentPage/ConnectedPagination.jsx

```diff
@@ -0,0 +1,31 @@
+import React, {
+  useEffect, useContext,
+} from 'react';
+import PropTypes from 'prop-types';
+import { connectPagination } from 'react-instantsearch-dom';
+import {
+  DataTable, DataTableContext,
+} from '@edx/paragon';
+
+export const BaseConnectedPagination = ({
+  currentRefinement: algoliaPageNo,
+  refine,
+}) => {
+  const { state: tableState } = useContext(DataTableContext);
+  useEffect(() => {
+    if (algoliaPageNo && tableState.pageIndex + 1 !== algoliaPageNo) {
+      // algolia is 1 indexed and react-table is 0 indexed
```

Comment:
To discuss: docs imply that Algolia is _not_ 1-indexed. https://www.algolia.com/doc/api-reference/api-parameters/page/#usage-notes

Reply:
I'm no longer using this component.